unifi/unifi.go

```diff
@@ -112,7 +112,7 @@ func (c *Client) setAPIUrlStyle(ctx context.Context) error {
 	defer resp.Body.Close()
 	_, _ = io.Copy(ioutil.Discard, resp.Body)
 
-	if resp.StatusCode == http.StatusOK {
```

Comment:
FWIW, our current implementation is consistent with another Unifi Go library: https://github.com/unpoller/unifi/blob/master/unifi.go#L193